tests/integration/examples/test_notebooks_tf2.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
one thing that is missing are the unit tests of the classes themselves. See for example: https://github.com/microsoft/recommenders/blob/main/tests/unit/recommenders/models/test_deeprec_model.py